manifests/pipecd/templates/envoy-configmap.yaml

```diff
@@ -61,21 +61,39 @@ data:
                     expose_headers: custom-header-1,grpc-status,grpc-message
 {{- end }}
                   routes:
+                    - match:
+                        prefix: /grpc.service.pipedservice.PipedService/
+                        grpc:
+                      route:
+                        cluster: grpc-piped-service
                     - match:
                         prefix: /pipe.api.service.pipedservice.PipedService/
                         grpc:
                       route:
                         cluster: grpc-piped-service
+                        prefix_rewrite: /grpc.service.pipedservice.PipedService/
```

Comment:
This allows the new control-plane can be able to accept gRPC requests from both old and new Piped agents.